web_sdk/libraries.json

```diff
@@ -0,0 +1,236 @@
+    "libraries": {
+      "async": {
+        "patches": "lib/_internal/js_runtime/lib/async_patch.dart",
+        "uri": "lib/async/async.dart"
+      },
+      "_interceptors": {
+        "uri": "lib/_internal/js_runtime/lib/interceptors.dart"
+      },
+      "mirrors": {
+        "patches": "lib/_internal/js_runtime/lib/mirrors_patch_cfe.dart",
+        "supported": false,
+        "uri": "lib/mirrors/mirrors.dart"
+      },
+      "_js_embedded_names": {
+        "uri": "lib/_internal/js_runtime/lib/shared/embedded_names.dart"
+      },
+      "io": {
+        "patches": "lib/_internal/js_runtime/lib/io_patch.dart",
+        "supported": false,
+        "uri": "lib/io/io.dart"
+      },
+      "_internal": {
+        "patches": "lib/_internal/js_runtime/lib/internal_patch.dart",
+        "uri": "lib/internal/internal.dart"
+      },
+      "_metadata": {
+        "uri": "lib/html/html_common/metadata.dart"
+      },
+      "_async_await_error_codes": {
+        "uri": "lib/_internal/js_runtime/lib/shared/async_await_error_codes.dart"
+      },
+      "_http": {
+        "uri": "lib/_http/http.dart"
+      },
+      "_js_primitives": {
+        "uri": "lib/_internal/js_runtime/lib/js_primitives.dart"
+      },
+      "_js_helper": {
+        "uri": "lib/_internal/js_runtime/lib/js_helper.dart"
+      },
+      "_chrome": {
+        "uri": "lib/_chrome/dart2js/chrome_dart2js.dart"
+      },
+      "html_common": {
+        "uri": "lib/html/html_common/html_common_dart2js.dart"
+      },
+      "ui": {
+        "uri": "../lib/ui/ui.dart"
+      },
+      "js": {
+        "uri": "lib/js/dart2js/js_dart2js.dart"
+      },
+      "_native_typed_data": {
+        "uri": "lib/_internal/js_runtime/lib/native_typed_data.dart"
+      },
+      "_js_names": {
+        "uri": "lib/_internal/js_runtime/lib/js_names.dart"
+      },
+      "core": {
+        "patches": "lib/_internal/js_runtime/lib/core_patch.dart",
+        "uri": "lib/core/core.dart"
+      },
+      "collection": {
+        "patches": "lib/_internal/js_runtime/lib/collection_patch.dart",
+        "uri": "lib/collection/collection.dart"
+      },
+      "js_util": {
+        "uri": "lib/js_util/dart2js/js_util_dart2js.dart"
+      },
+      "typed_data": {
+        "patches": "lib/_internal/js_runtime/lib/typed_data_patch.dart",
+        "uri": "lib/typed_data/typed_data.dart"
+      },
+      "web_audio": {
+        "uri": "lib/web_audio/dart2js/web_audio_dart2js.dart"
+      },
+      "html": {
+        "uri": "lib/html/dart2js/html_dart2js.dart"
+      },
+      "isolate": {
+        "patches": "lib/_internal/js_runtime/lib/isolate_patch.dart",
+        "supported": false,
+        "uri": "lib/isolate/isolate.dart"
+      },
+      "developer": {
+        "patches": "lib/_internal/js_runtime/lib/developer_patch.dart",
+        "uri": "lib/developer/developer.dart"
+      },
+      "web_gl": {
+        "uri": "lib/web_gl/dart2js/web_gl_dart2js.dart"
+      },
+      "indexed_db": {
```

Comment:
Curious what would happen if we removed some of the libraries we know we're not using.